lib/fluent/plugin/exception_detector.rb

```diff
@@ -59,7 +59,7 @@ def self.supported
     ].freeze
 
     PYTHON_RULES = [
-      rule(:start_state, /Traceback \(most recent call last\)/, :python),
+      rule(:start_state, /^Traceback \(most recent call last\):$/, :python),
```

Comment:
After adding test cases, the regexps seem to be satisfactory in the ruby / php test scenarios. I will try to come up with a test scenario that will cause the current regexp to fail before adding anchors without being able to validate.